l10n_it_codici_carica/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+====================================
+Italian Localization - codici carica
+====================================
+
+Tabella dei codici carica da usare nei dichiarativi fiscali italiani
+
+Configuration
+=============
+
+La tabella viene aggiunta nell'area contabilità, ed è raggiungibile nella 
+sezione configurazione ad chi ha i privilegi di account manager
+
+Usage
+=====
+
+Installando il modulo vengono precaricati i codici disponibili. 
+E' possibile aggiungere nuovi codici o modificare quelli esistenti.
+Non è possibile avere più di un elemento con lo stesso codice.
+
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/l10n-italy/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Alessandro Camilli <alessandrocamilli@openforce.it>
+
```

Comment:
Please add Funders section and add Odoo Italia Network to it.
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#funders